### PR TITLE
Don't depend on `stringof` in `mir.parse` test

### PR DESCRIPTION
**Pull request by dkorpel**
This is blocking https://github.com/dlang/dmd/pull/14218

Depending on the specific output of `.stringof` is bad practice, like the spec says:

> Implementation Defined: The string representation for a type or expression can vary.

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by dkorpel** [comment]
Can a new tag be made so buildkite picks it up?

**Comment by thewilsonator** [comment]
Yes, I'll leave that to @9il

**Comment by 9il** [comment]
Doesn't [v3.14.10](https://github.com/libmir/mir-algorithm/releases/tag/v3.14.10) contain this change?

**Comment by dkorpel** [comment]
Shortly after this was merged buildkite was still failing, and I was looking at releases and the latest one is from 2020 so I thought nothing changed. But it seems buildkite is green now, thanks!

**Pull request merged**
